### DISCUSSION TITLE
Multi Timbrality?

### DISCUSSION THREAD

**Discussion by rossec** [probonopd/MiniDexed#177] [Q&A] [answered]
Are there any plans to have each channel of minidexed play independently? This is similar to how the FB01 worked.

**Comment by craigyjp** [comment, marked as answer]
Isn't that the way it works now, just set each TG to a different midi channel. Its 8 part multitimbral.

**Comment by eLuSiVeMiTe** [comment]
> Are there any plans to have each channel of minidexed play independently? This is similar to how the FB01 worked.

Edit the performance file. Set midi from 255 (global) or 0 (off) to midi chanels of choice. 
Can also do it from the UI 
Just finished a 3 track test. 

All but drums are from minidexed 
https://youtu.be/-jcrZZTeDKY

**Reply by probonopd** [reply]
If you want the change temporary, just set another MIDI channel than "Omni" for each Tone Generator (TG) from the LCD menu.
If you want the change permanent, edit `performance.ini` (or Save -> Performance) from the LCD menu.

**Reply by probonopd** [reply]
Quite the studio @eLuSiVeMiTe. Do you have a 3D printer?

**Reply by eLuSiVeMiTe** [reply]
Cheers. 

It's been on my list for a few years now. I seem to keep missing out when a second hand unit comes up for grabs. 

 I had a fun lockdown building that up. Was purely DAW based until then.  

I'm still debating opening up the controller and fitting it internally. Or possibly just the screen and encoder.

**Reply by rossec** [reply]
thanks for the quick replys! I'll give this a shot. I don't have an lcd or rotary encoder so I didn't know this was possible. thanks again!

**Reply by rossec** [reply]
> If you want the change temporary, just set another MIDI channel than "Omni" for each Tone Generator (TG) from the LCD menu. If you want the change permanent, edit `performance.ini` (or Save -> Performance) from the LCD menu.

should this be more clear on the github homepage? I had no idea as it's not mentioned at all, and I don't have an obvious way of knowing without an LCD. This is a HUGE feature that really takes iit to another level!

**Reply by eLuSiVeMiTe** [reply]
![Screenshot_20220426-211641](https://user-images.githubusercontent.com/103902615/165384837-8a57ff5c-5216-47ec-885f-b98c0b046474.png)

**Reply by probonopd** [reply]
If I may have a wish, please link to the respective information instead of posting giant screenshots from a mobile phone. Thanks!

**Reply by eLuSiVeMiTe** [reply]
> If I may have a wish, please link to the respective information instead of posting giant screenshots from a mobile phone. Thanks!

You tell me how I can link to a paragraph midway down a very very long page of text and il ablige. :)  

 

It's in there somewhere. 

https://github.com/probonopd/MiniDexed/wiki/Menu 

The point was to show the information is present.

**Reply by probonopd** [reply]
Ah, right ;) Thanks!

**Comment by craigyjp** [comment]
Edit the channels with the encoder and then just save as a performance.
